lib/Web/API.hs

```diff
@@ -8,12 +8,38 @@ import Backend.Vault.Kv
 import CLI.Types
 import Coffer.Directory (Directory)
 import Coffer.Path (EntryPath, Path, QualifiedPath)
-import Data.HashMap.Strict
+import Data.Aeson.Casing (aesonPrefix, camelCase)
+import Data.Aeson.TH (deriveJSON)
+import Data.HashMap.Internal.Strict (HashMap)
```

Comment:
We should avoid importing `Internal` modules (there's no guarantee of backwards-compatibility, they can be modified or deleted altogether in between package versions).

Sometimes importing `Internal` modules, and taking on those risks, is necessary. But it should be avoided.

```suggestion
import Data.HashMap.Strict (HashMap)
```

Reply:
Ok. Thanks, I will pay attention to it.